src/app/styles/variables.scss

```diff
@@ -41,8 +41,8 @@ $color-dark: #343a40; //$gray-800 in Bootstrap4
 $color-primary-nav: #38598a;  // This is new from Bootstrap 4 theme
 $color-secondary-nav: #5475a7; // This is new too Bootstrap 4 theme
 $gray-lighter:#eee;
-$color-white: #fcfcfc; //Just shy of true whit.
-$white: #fcfcfc; //Just shy of true whit.
+$color-white: #ffffff; //Just shy of true whit.
+$white: #ffffff; //Just shy of true whit.
```

Comment:
Does this affect the page background color? If so, the page background should still be `#fcfcfc`.

Reply:
I've overwritten the background color of the form-control in my latest push. Let me know if there's a better way to do it. Thanks for the comment, Harry.